frictionless_ckan_mapper/ckan_to_frictionless.py

```diff
@@ -111,6 +111,8 @@ def dataset(ckandict):
     if 'tags' in ckandict:
         outdict['keywords'] = [tag['name'] for tag in ckandict['tags']]
         del outdict['tags']
+    if outdict.get("keywords") == []:
+        del outdict["keywords"]
```

Comment:
won't this error is `keywords` is not in the dict ... (or is it always in the dict?)

Reply:
No, this is why I did it this way:

```
>>> outdict = {}
>>> if not outdict.get("keywords"):
...     del outdict["keywords"]
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
KeyError: 'keywords'

>>> if outdict.get("keywords") == []:
...     del outdict["keywords"]
...
>>>
```